### DISCUSSION TITLE
position! mode color switch theme

### DISCUSSION THREAD

**Discussion by fernandogprieto** [facebook/docusaurus#8861] [Q&A] [answered]
Hello Guys, I would like to change position my mode color switch theme before the search bar.

My site:
![image](https://user-images.githubusercontent.com/113651560/229336572-204cf6ab-c548-4772-b08c-58c6c45c1b44.png)

Expected:
![image](https://user-images.githubusercontent.com/113651560/229336711-75301835-5192-4b98-85ae-1762dcca4fbd.png)
I would like to configure my site with this setup.

**Comment by slorber** [comment, marked as answer]
The color mode switch is already before the search (cf the Docusaurus site itself)

If it's not the case then it's likely because you declared a custom item of type "search" in navbar items: try to remove it and you should get the same result as on the Docusaurus site. (I understand how it can be confusing due to historical reasons)

For more fancy positions you can swizzle `Navbar/Content`

**Reply by fernandogprieto** [reply]
Thank you for your time. It's fix it.
